pixleesdk/src/main/java/com/pixlee/pixleesdk/network/NetworkModule.java

```diff
@@ -0,0 +1,198 @@
+package com.pixlee.pixleesdk.network;
```

Comment:
are these network files created by us or imported in?

Reply:
This class is about created by me. PXLClient.java is for Our client and NetworkModule.java is for us to wrap Retrofit codes.